src/sentry/static/sentry/app/components/events/meta/annotatedText.tsx

```diff
@@ -107,26 +112,24 @@ function renderErrors(errors) {
   );
 }
 
-const AnnotatedText = ({value, chunks, errors, remarks, ...props}) => (
-  <span {...props}>
-    {renderValue(value, chunks, errors, remarks)} {renderErrors(errors)}
-  </span>
-);
-
-AnnotatedText.propTypes = {
-  value: PropTypes.node,
-  chunks: PropTypes.array,
-  errors: PropTypes.array,
-  remarks: PropTypes.array,
-};
-
-AnnotatedText.defaultProps = {
-  chunks: [],
-  errors: [],
-  remarks: [],
-};
+class AnnotatedText extends React.Component<Props, {}> {
```

Comment:
```suggestion
class AnnotatedText extends React.Component<Props> {
```

Reply:
But since this function isn't using any state you could convert it to arrow function 😉

Reply:
it will have state soon